trueos-handbook/install.rst

```diff
@@ -9,6 +9,290 @@ graphical desktop directly onto a hard drive or into a virtual machine
 using virtualization software such as
 `VirtualBox <https://www.virtualbox.org/>`_.
 
+.. index:: Download and Prepare to Install
+.. _Download and Prepare to Install:
+
+Download and Prepare to Install
+===============================
+
+|trueos| uses a rolling release model rather than versioned releases.
+There are two primary options of |trueos| install: STABLE and UNSTABLE:
+
+1. STABLE is meant to be less frequently updated and synchronized with
```

Comment:
We need to dissassociate the words "updated" and "synchronized": one is used negatively and the other positively (less updated but more synchronized). Maybe split into two sentences or just better explain the difference?